### ISSUE TITLE
Unused application property hazelcast.backup-count

### ISSUE THREAD

**Issue by Akuka** [jhipster/generator-jhipster#7491]
##### **Overview of the issue**

There is no use of the application property jhipster.cache.hazelcast.backup-count

##### **Suggest a Fix**

Not sure if the fix should be here:
https://github.com/jhipster/generator-jhipster/blob/master/generators/server/templates/src/main/java/package/config/CacheConfiguration.java.ejs#L277
or at: 
https://github.com/jhipster/generator-jhipster/blob/master/generators/server/templates/src/main/java/package/config/CacheConfiguration.java.ejs#L300

##### **JHipster Version(s)**

4.14.3 / 5.0.0-beta

##### **Browsers and Operating System**

- [V ] Checking this box is mandatory (this is just to show you read everything)

**Comment by pmverma** [comment]
Yes, I also confirm that value from `yml` is not being applied as per my generated code.

**Comment by jdubois** [comment]
I remember adding this when I did the first implementation, but this has been refactored (the contribution was from Hazelcast themselves!!), so it's probably unused now.
Let's remove it - would you like to do the PR so you are the author of the change?

**Comment by Akuka** [comment]
OK, I will do a PR

**Comment by deepu105** [comment]
I wouldn't remove it, rather I would use the value instead of the hard
coded ones, I noticed this earlier but forgot to fix it

**Comment by Akuka** [comment]
@deepu105, for which configurations this property related, this:
https://github.com/jhipster/generator-jhipster/blob/master/generators/server/templates/src/main/java/package/config/CacheConfiguration.java.ejs#L277
or at:
https://github.com/jhipster/generator-jhipster/blob/master/generators/server/templates/src/main/java/package/config/CacheConfiguration.java.ejs#L300
?
